### PR TITLE
Revert "[SIG-3211] Enable typescript"

### PR DESCRIPTION
**Pull request by aditudorache**
Reverts Amsterdam/signals-frontend#1188

### CODE REVIEW

**Comment by jpoppe** [comment]
Thanks, great work BTW, let's pick this up next week.

File: tsconfig.json [hunk 1]
```diff
@@ -1,35 +0,0 @@
-{
```
Comment (MauritsR) [inline tsconfig.json:1]: This file should be added to the COPY command of the Dockerfile

**Comment by boris-arkenaar** [comment]
Not working correctly yet?

**Comment by jpoppe** [comment]
@boris-arkenaar at least I would like to review it ;)

**Comment by aditudorache** [comment]
Decided to postpone it a bit. :-). Sorry for the inconvenience

**Review by jpoppe — APPROVED** [review]

**Pull request merged**